app/models/organization.rb

```diff
@@ -15,6 +15,13 @@ class Organization < ActiveRecord::Base
 
   paginates_per 20
 
+  include PgSearch
+    pg_search_scope :search,
+      :against => :name,
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.